private/bufpkg/bufstudioagent/plain_post_handler.go

```diff
@@ -222,6 +203,43 @@ func (i *plainPostHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func connectClientOptionsFromContentType(contentType string, requestMsgSize int) ([]connect.ClientOption, error) {
+	switch contentType {
+	case "application/grpc", "application/grpc+proto":
+		return []connect.ClientOption{
+			connect.WithGRPC(),
+			connect.WithCodec(&bufferCodec{name: "proto"}),
+		}, nil
+	case "application/grpc-json":
```

Comment:
I've never seen this before, shouldn't this be `application/grpc+json`?

Reply:
Uh, yes, it should be 😬